doc/coq.rst

```diff
@@ -104,9 +110,11 @@ The semantics of the fields are:
   theory, which means the ``Coq`` prefix won't be bound, and ``Coq.Init.Prelude``
   won't be imported by default.
 
-- The path to the installed locations of the ``<ocaml_plugins>`` is passed to
-  ``coqdep`` and ``coqc`` using Coq's ``-I`` flag. This allows a Coq theory to
-  depend on OCaml plugins.
+- If ``plugins`` is present, Dune will pass the corresponding flags to
```

Comment:
```suggestion
- If the ``plugins`` field is present, Dune will pass the corresponding flags to
```

Reply:
Note both styles appear in the doc, we should pick one. "If ``package`` is present ..."